<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExifStatistics v1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_10">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Settings</string>
          </property>
          <property name="flat">
           <bool>false</bool>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Folder:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="3">
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLineEdit" name="txtFolderName">
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btnBrowse">
                 <property name="text">
                  <string>Browse</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>File Format:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="sizeConstraint">
                <enum>QLayout::SetDefaultConstraint</enum>
               </property>
               <item>
                <widget class="QCheckBox" name="chkJPEG">
                 <property name="text">
                  <string>JPEG</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkTIFF">
                 <property name="text">
                  <string>TIFF</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="chkRAW">
                 <property name="text">
                  <string>RAW</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item row="1" column="3">
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Filter</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Make:</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="txtMake"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Model:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="txtModel"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="btnStart">
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QProgressBar" name="progressBar">
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>0</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Pictures analysed:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lblPicturesFound">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Focal Length</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QTreeWidget" name="treeFocal">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>mm</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeFocalSorted">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>mm</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_5">
         <attribute name="title">
          <string>Focal Length (35mm)</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QTreeWidget" name="treeFocal35">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>mm</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeFocal35Sorted">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>mm</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Aperture</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QTreeWidget" name="treeAperture">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>f/</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeApertureSorted">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>f/</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>ISO</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QTreeWidget" name="treeISO">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>ISO</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeISOSorted">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>ISO</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Exposure Time</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QTreeWidget" name="treeExposure">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>Seconds</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeExposureSorted">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>75</number>
              </attribute>
              <column>
               <property name="text">
                <string>Seconds</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_6">
         <attribute name="title">
          <string>Camera</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_11">
          <item row="0" column="0">
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QTreeWidget" name="treeCamera">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>100</number>
              </attribute>
              <column>
               <property name="text">
                <string>Make &amp; Model</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
            <item>
             <widget class="QTreeWidget" name="treeCameraSorted">
              <property name="rootIsDecorated">
               <bool>false</bool>
              </property>
              <attribute name="headerDefaultSectionSize">
               <number>100</number>
              </attribute>
              <column>
               <property name="text">
                <string>Make &amp; Model</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>Pictures</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
              <column>
               <property name="text">
                <string>%</string>
               </property>
               <property name="textAlignment">
                <set>AlignHCenter|AlignVCenter|AlignCenter</set>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QPushButton" name="btnExport">
          <property name="text">
           <string>Export</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnClose">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
